File: app/src/main/java/online/motohub/activity/ComingSoonActivity.java
package online.motohub.activity;

import android.os.Bundle;
import android.support.v7.widget.Toolbar;
import android.view.View;

import butterknife.BindString;
import butterknife.BindView;
import butterknife.ButterKnife;
import butterknife.OnClick;
import online.motohub.R;

public class ComingSoonActivity extends BaseActivity {

    @BindView(R.id.toolbar)
    Toolbar mToolbar;
    @BindString(R.string.coming_soon)
    String mToolbarTitle;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_comming_soon);

        ButterKnife.bind(this);

        initViews();

    }

    private void initViews() {

        setToolbar(mToolbar, mToolbarTitle);

        showToolbarBtn(mToolbar, R.id.toolbar_back_img_btn);
    }

    @Override
    public void onBackPressed() {
        finish();
    }

    @OnClick({R.id.toolbar_back_img_btn})
    public void onClick(View v) {
        switch (v.getId()) {
            case R.id.toolbar_back_img_btn:
                finish();
                break;
        }
    }
}

File: app/src/main/java/online/motohub/newdesign/activity/ComingSoonActivity.kt
package online.motohub.newdesign.activity

import android.os.Bundle
import android.view.View
import online.motohub.R
import online.motohub.activity.BaseActivity

class ComingSoonActivity : BaseActivity(), View.OnClickListener {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_comming_soon)
    }

    override fun onBackPressed() {
        finish()
    }

    override fun onClick(v: View?) {
        when (v!!.id) {
            R.id.backIcon -> {
                finish()
            }
        }
    }

}
